roles/cs.redis/tasks/install.yml

```diff
@@ -1,4 +1,13 @@
-- name: Install redis packages
+- name: Uninstall redis packages
+  package:
+    name: >-
+      {{ 'valkey' if redis_valkey_flavor == 'redis' else 'redis' }}
```

Comment:
we such logic in lots of places, could we set something like `redis_uninstalling_flavor` or similar in defaults and compute it only once?
it would make rest of the implementation easier to read.